obal/data/roles/git_branch_and_commit/tasks/main.yml

```diff
@@ -0,0 +1,29 @@
+---
+- name: 'Get updated version from spec'
+  command: "rpmspec --query --queryformat=%{version} --srpm {{ spec_file_path }}"
+  args:
+    chdir: "{{ inventory_dir }}"
+  changed_when: true
+  register: updated_version
+
+- name: 'Get current git branch'
+  command: "git rev-parse --abbrev-ref HEAD"
+  args:
+    chdir: "{{ inventory_dir }}"
+  changed_when: true
+  register: git_branch
+  run_once: true
+
+- name: 'Create git branch'
+  command: "git checkout -b {{ git_branch.stdout }}-update-{{ inventory_hostname }}-{{ updated_version.stdout }}"
+  args:
+    chdir: "{{ inventory_dir }}"
+  run_once: true
+  changed_when: true
```

Comment:
why the explicit true here?

Reply:
Lint will fail because this is a command without any other indicator of changed. I can add checks for uncommitted files and what branch I am on to make it more idempotent. But I was trying to more re-organize the code rather than so much as change it heavily.

Reply:
interesting, thanks!